src/test/java/shop/VirtualItemTest.java

```diff
@@ -0,0 +1,17 @@
+package shop;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class VirtualItemTest {
+    private static final double SIZE = 20000.01;
+
+    @Test
+    void setSizeOnDisk() {
```

Comment:
VirtualItem class is designed as simple model without any business logic. It doesn't make sense to write tests for getters/setters. The only method, which can be covered with test, is toString in this case. This task has 2 correct resolutions:
- don't write tests
- write test for toString